scripts/ironic-common.sh

```diff
@@ -28,9 +28,9 @@ function wait_for_interface_or_ip() {
       sleep 1
     done
   else
-    until [ ! -z "${IRONIC_IP}" ]; do
+    until [ ! -z "${IRONIC_MY_IP}" ]; do
```

Comment:
why rename the var here?

Reply:
In the other PR, can you call the var IRONIC_SERVICE_IP instead? or just use IRONIC_URL_HOST for the service ip too.

Reply:
we need ens4 IP for file downloads. hence IRONIC_IP will be service IP and IRONIC_MY_IP will be ens4 IP. IRONIC_URL_HOST is getting set to IRONIC_IP only. changing IRONIC_IP to IRONIC_SERVICE_IP will be a big change.

Reply:
What about file downloads when the ironic pod moves to a different host then?